server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -126,6 +127,25 @@ public void putGcCandidates(TableId tableId, Collection<StoredTabletFile> candid
     }
   }
 
+  @Override
+  public void putGcFdCandidates(TableId tableId, Collection<String> candidates) {
+
+    if (RootTable.ID.equals(tableId)) {
+      // Directories are unexpected for the root tablet, so convert to stored tablet file
+      mutateRootGcCandidates(
+          rgcc -> rgcc.add(Collections2.transform(candidates, StoredTabletFile::new)));
```

Comment:
Can you use Java built in Stream to map to StoredTableFile, instead of adding the Guava usage?

Something like:
```suggestion
          rgcc -> rgcc.add(Collections2.transform(candidates, StoredTabletFile::new)));
          rgcc -> rgcc.add(candidates.stream().map(StoredTabletFile::new).collect(toList())));
```

I know it's slightly longer, but I think it's generally safer to use built-ins than to rely on Guava.

Reply:
I changed this in d259836 to use streams.  Made the method take an iterator instead of a collection. This allowed streams to be used w/o copying to a new collection like the Guava code and its still short.

Reply:
That works. :smiley_cat: